src/common/components/repository-row/index.js

```diff
@@ -296,7 +296,7 @@ class RepositoryRow extends Component {
         actionSpinner = true;
         actionText = 'Checking...';
       } else {
-        actionText = 'Register';
+        actionText = 'Register Name';
```

Comment:
@matthewpaulthomas This one for you. I was just following what was in the card but I had the same thought as @bartaz